### ISSUE TITLE
Documentation link on server admin page is broken in release_5.2.1

### ISSUE THREAD

**Issue by tst-lsavoie** [google/earthenterprise#671]
It appears the "create_links" function was not included in the RPM installers.

**Comment by tst-lsavoie** [comment]
Closed via PR #672.

**Comment by tst-ccamp** [comment]
There is still a bug with the RPMs. Looks like the links for /opt/google/log, etc., were added. But, the directives to create them as directories are still there. These Lines also need to be removed:
```
    directory('/opt/google/etc', 0775)
    directory('/opt/google/run', 0775)
    directory('/opt/google/log', 0775)
```
